README.md

```diff
@@ -26,10 +26,12 @@ We welcome contributions and feedback. Please follow our [contributing guideline
 
 ## First Time Setup
 
-1. Create a new Single Page App in [devconsole](https://app.laserfiche.com/devconsole/apps) and add the authentication redirect URI (e.g.: <https://myapp.example.com/lf-sample-app/>) which is also the root page of this app.
+0. Clone the repo
+1. Create a new Single Page App in devconsole for [US](https://app.laserfiche.com/devconsole/apps), [Canada](https://app.laserfiche.ca/devconsole/apps), or [Europe](https://app.eu.laserfiche.com/devconsole/apps) (depending on your region) and add the authentication redirect URI (e.g.: <https://myapp.example.com/lf-sample-app/>) which is also the root page of this app.
```

Comment:
does something like "which should be where you are hosting the root page of this app" make more sense to you than "which is also the root page of this app"? Thinking of a way to update that to avoid the confusion when you first started building these, that where you are hosting it must match the redirect uri which must match what is registered in devconsole

Reply:
I've updated it to "...add the redirect URI (...). The redirect URI in the devconsole must match the REDIRECT_URI variable in `src/app/config.ts`, which must match the host address.